kernel/userspace.c

```diff
@@ -301,6 +301,7 @@ void *z_impl_k_object_alloc(enum k_objects otype)
 	case K_OBJ_SYS_MUTEX:			/* Lives in user memory */
 	case K_OBJ_THREAD_STACK_ELEMENT:	/* No aligned allocator */
 	case K_OBJ_NET_SOCKET:			/* Indeterminate size */
+	case K_OBJ_NET_IF:			/* Indeterminate size */
```

Comment:
isn't this `struct net_if`? We know how big it is. Or is it always embedded within some other containing struct?

Reply:
> isn't this `struct net_if`? We know how big it is. Or is it always embedded within some other containing struct?

Good point, net_if is always the same size. I will send a new version.